code/Modules/HttpFS/private/emsc/emscURLLoader.cc

```diff
@@ -33,13 +33,14 @@ emscURLLoader::startRequest(const Ptr<IORead>& req) {
     // start the asynchronous XHR
     // NOTE: we can only load from our own HTTP server, so the host part of 
     // the URL is completely irrelevant...
-    String urlPath = req->Url.PathToEnd();
-    emscripten_async_wget_data(urlPath.AsCStr(), (void*) reqPtr, emscURLLoader::onLoaded, emscURLLoader::onFailed);
+    StringBuilder urlPath = "/";
```

Comment:
Hmm, I just noticed that starting with "/" breaks my github pages sample pages since it is no longer possible to define a relative URL (URLs would always start at the root). I'll revert this one line.